WooCommerce/Classes/ViewRelated/ReusableViews/BadgedLeftImageTableViewCell.swift

```diff
@@ -0,0 +1,93 @@
+import UIKit
+
+
+/// Represents a regular UITableView Cell: [Image | Text |  Disclosure]
+///
+class BadgedLeftImageTableViewCell: UITableViewCell {
```

Comment:
Just wondering: Wouldn't inherit like `class BadgedLeftImageTableViewCell: LeftImageTableViewCell` remove the need to declare `leftImage`, `labelText`, ... and we could have only the extensions to setup this view?